packages/recomponents/src/components/r-tabs/r-tabs.spec.js

```diff
@@ -126,14 +127,41 @@ describe('r-tabs.vue', () => {
                     fullPath: 'test-route',
                 },
                 $router: {
-                    resolve: route => ({route}),
+                    resolve: route => ({...route, href: 'test-route'}),
                 },
             },
         });
 
         expect(wrapper).toMatchSnapshot();
     });
 
+    it('should not try to go to the same route where we already are', async () => {
+        const localVue = createLocalVue();
+        localVue.use(VueRouter);
+        const route = {name: 'child', path: '/nested-route'};
+        const route1 = {name: 'child2', path: '/nested-route2'};
+        const router = new VueRouter({routes: [route, route1]});
+        router.push(route);
+        localVue.component('r-tab', RTab);
+        const routerPushSpy = jest.spyOn(router, 'push');
+
+        mount(RTabs, {
+            localVue,
+            router,
+            slots: {
```

Comment:
@Rebilly/js Note that here I used `slots` testing param instead of using `render`method like in the existing tests. I think that it's easier to write and read the code like this. Maybe we could consider to write/refactor tests from now on. What do you think?